src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+2. If you haven't already done so, complete the following:
+    ```
+    git clone git@github.com:newrelic/newrelic-opentelemetry-examples.git
+    cd newrelic-opentelemetry-examples/getting-started-guides/java/Uninstrumented
+    ```
+3. Build the demo app with this command:
+    ```
+    ./gradlew build
+    ```
+    <Callout variant="tip">
+      After a successful build, you will find the resulting application JAR file in `Uninstrumented/build/libs`.
+    </Callout>
+
+4. Export the following environment variables to configure the agent. Click on each for details:
+
+    <CollapserGroup>
+      <Collapser
+        className="freq-link"
+        id="exp-bucket-hist"
+        title="OTEL_EXPORTER_OTLP_HEADERS=api-key=YOUR_LICENSE_KEY"
+      >
+        Headers: This determines which New Relic account your data will be sent to. Replace `YOUR_LICENSE_KEY` with your [ingest license key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key).
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="temp"
+        title="OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317 (For the US)"
+      >
+        This is the endpoint your application telemetry will be exported to. Export data to New Relic's OTLP endpoint. If you're in the EU, use this endpoint:
+
+        OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.eu01.nr-data.net:4317
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="service-name"
+        title="OTEL_SERVICE_NAME=getting-started-java"
+      >
+        This is the name that associates your application telemetry with an entity in New Relic. In this tutorial we're calling it `getting-started-java`, but you could call it anything you want.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="instance-id"
+        title="OTEL_RESOURCE_ATTRIBUTES=service.instance.id=INSERT_YOUR_OWN_ID_HERE"
+      >
+        Replace `INSERT_YOUR_OWN_ID_HERE` with a unique instance id. For example, you could use `1234`. For more details, see OpenTelemetry's [Resource Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#service).
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="length-limit"
+        title="OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT=4095"
+      >
+        Setting a limit on the maximum attribute value length will prevent span data from getting dropped if it exceeds the [New Relic limit](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-attributes/).
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="temp2"
+        title="OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE=DELTA"
+      >
+        New Relic supports metrics in delta temporality instead of the default of cumulative.
```

Comment:
this was hard for me to parse. esp the 'metrics in delta temporality...' but maybe that's understood by OTel people

Reply:
@zuluecho9 this is a bit arcane, but I think we can eventually link to some new material we'll be adding about metrics that will also include histogram details.